src/test/unit/rc_controls_unittest.cc

```diff
@@ -445,6 +465,26 @@ static const adjustmentConfig_t rateProfileAdjustmentConfig = {
 TEST_F(RcControlsAdjustmentsTest, processRcRateProfileAdjustments)
 {
     // given
+    controlRateConfig_t controlRateConfig = {
+            .rcRate8 = 90,
+            .rcExpo8 = 0,
+            .thrMid8 = 0,
+            .thrExpo8 = 0,
+            .rates = {0,0,0},
+            .dynThrPID = 0,
+            .rcYawExpo8 = 0,
+            .tpa_breakpoint = 0
+    };
+
+    // and
+    memset(&rxConfig, 0, sizeof (rxConfig));
+    rxConfig.mincheck = DEFAULT_MIN_CHECK;
+    rxConfig.maxcheck = DEFAULT_MAX_CHECK;
+    rxConfig.midrc = 1500;
+
+    adjustmentStateMask = 0;
+    memset(&adjustmentStates, 0, sizeof(adjustmentStates));
+
     int adjustmentIndex = 3;
```

Comment:
duplicate test setup code can be removed.  See `RcControlsAdjustmentsTest::SetUp()`  It looks like there might be some duplicate configuration of controlRateConfig anyway.